canonical-web-and-design/canonicalwebteam.blog
# v3.1.0: Allow setting feed_description through build_blueprint

Fixes https://github.com/canonical-web-and-design/canonicalwebteam.blog/issues/108

QA
--

QA with https://github.com/canonical-web-and-design/ubuntu.com/pull/5784 - go to https://ubuntu-com-canonical-web-and-design-pr-5784.run.demo.haus/blog/feed, open the feed, see the `<description>` says "The Ubuntu Blog Feed".
